addons/pinniped/config-controller/controllers/controller_test.go

```diff
@@ -42,215 +44,166 @@ var _ = Describe("Controller", func() {
 				},
 			},
 		}
+
+		pinnipedCBSecret = &corev1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: pinnipedNamespace,
+				Name:      fmt.Sprintf("%s-%s-package", cluster.Name, "pinniped.tanzu.vmware.com"),
+				Labels: map[string]string{
+					constants.PackageNameLabel:    TestPinnipedLabel,
+					constants.TKGClusterNameLabel: cluster.Name,
+				},
+			},
+			Type: constants.ClusterBootstrapManagedSecret,
+		}
 	})
 
-	Context("Cluster", func() {
+	Context("ClusterBootstrap Secret", func() {
 		BeforeEach(func() {
 			create(ctx, cluster)
+			create(ctx, pinnipedCBSecret)
 		})
 
 		AfterEach(func() {
-			delete(ctx, cluster)
-		})
-
-		When("cluster is created", func() {
-			It("creates a secret with identity_management_type set to none", func() {
-				Eventually(addonSecretFunc(ctx, cluster, nil)).Should(Succeed())
-			})
+			deleteObject(ctx, cluster)
```

Comment:
since we did `delete()` -> `deleteObject()`, should we also do `create()` -> `createObject()`?

Reply:
sure, I had to change `delete` to `deleteObject` because I was using the actual `delete` (for removing a key from a map), but I can change `create` as well.